### PR TITLE
Removes CMS_TOKEN

### LINKED ISSUE #297: Remove PLIO_CMS_TOKEN from environment variables

### PR DESCRIPTION
**Pull request by Salonichoudhary07**
Fixes #297 

## Summary

CMS_TOKEN was not being used at the backend.

## Test Plan

- [ ] <strike>Wrote tests</strike>
- [x] Tested locally
- [ ] Tested on staging
- [ ] Tested on production
- [x] If adding or removing any environment variable:
    - [x] update `docs/ENV.md`
    - [ ] update the environment variables for staging and production
- [ ] <strike>If changes in DB, update [DB schema] and BigQuery (staging and prod)</strike>

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # django
 APP_ENV='local' # local, staging, production
-SECRET_KEY='' # generate from https://djecrety.ir/
+SECRET_KEY='i%&w001-tkaz_f&r#=1%9^90eaqdhu1byjjx#11n*zf&ua@s9@' # generate from https://djecrety.ir/
```
Comment (deepansh96) [inline .env.example:3]: I think you forgot to remove this value.
This secret key's value should not be present in `.env.example` file. This file is supposed to be just an example. Everyone's secret key would be different and stored in their personal `.env` file

File: docs/ENV.md [hunk 2]
```diff
@@ -83,9 +83,7 @@ Hostname of your Redis instance
 #### `REDIS_PORT`
 Port of your Redis instance
 
-### Plio CMS
-#### `CMS_TOKEN`
-Secret token for Avanti's [Content Management System](http://cms.peerlearning.com) - if you plan to use it, you'll need to ask for access from the admins.
+
```
Comment (deepansh96) [inline docs/ENV.md:86]: Extra line

File: .env.example [hunk 3]
```diff
@@ -37,8 +37,7 @@ BIGQUERY_CREDENTIALS=''
 REDIS_HOSTNAME='redis'
 REDIS_PORT=6379
 
-# plio cms token
-CMS_TOKEN=''
+
```
Comment (deepansh96) [inline .env.example:40]: Extra line

**Review by deepansh96 — CHANGES_REQUESTED** [review]
Minor changes

**Review by dalmia — APPROVED** [review]

**Pull request merged**